.github/workflows/test-environment.yml

```diff
@@ -0,0 +1,127 @@
+name: sanity workflow
+
+on:
+  # Ability to execute on demand
+  workflow_dispatch:
+    inputs:
+      deployment_name:
+        type: string
+        description: |
+          Name your environment (Only a-zA-Z0-9 and `-`).
+          For example: john-8-7-2-June01'
+        required: true
+      elk-stack-version:
+        required: true
+        description: "Elastic Cloud stack version"
+        default: "8.7.2-SNAPSHOT"
+      ess-region:
+        required: true
+        description: "Elastic Cloud deployment region"
+        default: "gcp-us-west2"
+      cleanup-env:
+        description: "Boolean: Cleanup resources after provision"
+        default: false
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_REGION: "eu-west-3"
+  TF_VAR_ec_api_key: ${{ secrets.CI_EC_API_KEY }}
+  WORKING_DIR: deploy/test-environments
+
+jobs:
+  Setup:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 15
+    outputs:
+      stack-version: ${{ steps.set-input-vars.outputs.stack-version }}
+      ess-region: ${{ steps.set-input-vars.outputs.ess-region }}
+      deployment-name: ${{ steps.set-input-vars.outputs.deployment-name }}
+      tf-folder-date: ${{ steps.gen-tf-name.outputs.tf-folder-date}}
```

Comment:
I don't understand this part, why do we need it?

Reply:
Alright, I will remove the Setup job and merge the input variables into the Deploy job.